app/models/signup.rb

```diff
@@ -34,7 +34,7 @@ def self.send_message(message_name, batch_size=nil, after: nil)
       end
 
       if signup.phone_number.present?
-        TwilioService.send_text_message(to: signup.phone_number, body: message.sms_body)
+        TwilioService.new(:ctc).send_text_message(to: signup.phone_number, body: message.sms_body)
```

Comment:
this is actually not just a CTC thing! see: https://www.getyourrefund.org/en/sign-up

Reply:
that makes me moderately unhappy. but it's kinda sorta ok as right now ctc and gyr seem to be sharing the same twilio setup.